CHANGELOG.md

```diff
@@ -148,3 +148,7 @@ All notable changes to this project will be documented in this file. The format
 ### Changed:
  - Deprecate `subnetIP` and `subnetPrefix` in values.yaml, in favor of `cidr` field within `vipNetworkList`.
  - Update `spec.network` to include `networkName` and `cidr` information in AviInfraSetting CRD.
+
+### Known Issues
```

Comment:
We are providing `Bugs Fixed` as the heading

Reply:
This is current on-going issue which will be fixed in later release. So mentioned as Known issues.

Reply:
ah got it. Wondering if we should mention this here. Is it better to just open an issue on github for this?